src/battle/battle_2.c

```diff
@@ -1153,14 +1144,14 @@ void c2_8011A1C(void)
 
     for (i = 0; i < 80; i++)
     {
-        gUnknown_03004DE0.unk0[i] = 0xF0;
-        gUnknown_03004DE0.unk780[i] = 0xF0;
+        gUnknown_03004DE0[0][i] = 0xF0;
+        gUnknown_03004DE0[1][i] = 0xF0;
     }
     for (i = 80; i < 160; i++)
     {
         asm(""::"r"(i));  // Needed to stop the compiler from optimizing out the loop counter
```

Comment:
Is this asm hack still needed? Sometimes, I have to do stuff like this when I don't have the format quite right.

Reply:
Yes. I tried to fix it, but I still can't get it to match without that.